lib/tasks/attempts.rake

```diff
@@ -11,17 +11,24 @@ namespace :attempts do
     resp = conn.post('/api/irs_attempts_api/security_events', body) do |req|
       req.headers['Authorization'] =
         "Bearer #{IdentityConfig.store.irs_attempt_api_csp_id} #{auth_token}"
-    end.body
-
-    events = JSON.parse(resp)
+    end
+    encrypted_data = Base64.strict_decode64(resp.body)
+    iv = Base64.strict_decode64(resp.headers['x-payload-iv'])
+    encrypted_key = Base64.strict_decode64(resp.headers['x-payload-key'])
+    private_key = OpenSSL::PKey::RSA.new(File.read(private_key_path))
+    key = private_key.private_decrypt(encrypted_key)
+    decrypted = IrsAttemptsApi::EnvelopeEncryptor.decrypt(
+      encrypted_data: encrypted_data, key: key,
+      iv: iv
+    )
```

Comment:
same re: all same or all different
```suggestion
    decrypted = IrsAttemptsApi::EnvelopeEncryptor.decrypt(
      encrypted_data: encrypted_data, key: key, iv: iv,
    )
```

Reply:
Good call, fixed in https://github.com/18F/identity-idp/pull/6776/commits/a4f3e02b0f813777a80f4795273c149bb46003f0